src/app/core/services/label.service.ts

```diff
@@ -6,6 +6,15 @@ import { Observable } from 'rxjs';
 import { SEVERITY_ORDER } from '../../core/models/issue.model';
 import { User } from '../models/user.model';
 
+/* The threshold to decide if color is dark or light.
+A higher threshold value will result in more colors determined to be "dark".
+W3C recommendation is 0.179, but 0.184 is chosen so that some colors (like bright red)
+have white text instead of black. Github labels are also red/white instead of red/black */
```

Comment:
This line can be replaced by the following, to avoid referring to text color (as the constant is not about text color)
`are considered dark (Github too consider them dark)`